src/util.py

```diff
@@ -260,12 +265,115 @@ def xr_add_cyclic_longitudes(da, coord):
 
     return new_da
 
+def set_vector_density(data, lat_density=1, lon_density=1, minDistance=0, otherVars=[]):
+    """
+    Utility function to change density of vector plots.
+
+    Ars:
+
+        data (:class:`xarray.core.dataarray.DataArray`):
+            Data array that contains the vector plot latitude/longitude data.
+
+        lat_density (:class:`int`):
+            Value in range (0,1] that determines the density of the vectors in the y range.
+
+        lon_density (:class:`int`):
+            Value in range (0,1] that determines the density of the vectors in the x range.
+
+        minDistance (:class:`int`):
+            Value in degrees that determines the distance between the vectors.
+        
+        otherVars (:class: array of `xarray.core.dataarray.DataArray`):
+            Other variables that require the same density reduction as the lat/lon data
+
+    """
+    import math
+    import warnings
+
+    if minDistance != 0 and (lat_density != 1 or lon_density != 1):
+        raise Exception("minDistance and lat/lon_density parameters cannot be used simulatenously.")
```

Comment:
This constraint should be documented in the beginning of the function (i.e. docstrings). Indeed, it seems to me that having either minDistance or lat/lon density pair as input arguments would be enough, as already suggested by @hCraker. I could not find further dicsussion on this; could you clarify?

Reply:
Agreed, minDistance is probably enough. I thought originally that lat/lon_density would be useful if the user didn't know the distance they wanted between the vectors, but I took those arguments out.